bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-dropdown/component.jsx

```diff
@@ -151,6 +151,7 @@ class UserDropdown extends Component {
           iconRight,
         }}
         className={key === this.props.getEmoji ? styles.emojiSelected : null}
+        data-test={key}
```

Comment:
How is this random value of `data-test` helping us? Wouldn't it be more useful if we prefix-ed it with some string?

Reply:
I'm not sure about that either, but what really help are the aria-labels. We could use more of those because it's easier to aim for the selectors in puppeteer